aws/websites/metrics.pytorch.org/update_dashboards.py

```diff
@@ -0,0 +1,85 @@
+"""
+This script pulls down dashboards from Grafana, matches them by name to their
+correpsonding JSONs in files/dashboards, and sends a new commit to
+pytorch/test-infra if anything has changed
+"""
+
+import os
+import requests
+import json
+import difflib
+from pathlib import Path
+from typing import List, Dict, Any, Tuple
+
+user = os.environ["GRAFANA_USER"]
+password = os.environ["GRAFANA_PASSWORD"]
+
+Dashboard = Dict[str, Any]
+
+ROOT = Path(__file__).resolve().parent
+DASHBOARD_DIR = ROOT / "files" / "dashboards"
+
+
+def grafana(url):
+    base = "https://metrics.pytorch.org/api"
+    r = requests.get(f"{base}/{url.lstrip('/')}", auth=(user, password))
+    value = r.json()
+    if isinstance(value, dict) and value.get("message", None) is not None:
+        raise RuntimeError(value)
+    return value
+
+
+def get_dashboards() -> List[Dashboard]:
+    response = grafana("search?query=")
+    return [item for item in response if item.get("type") == "dash-db"]
+
+
+def files_by_uid() -> List[Dict[str, Any]]:
```

Comment:
this seems wrong (should be just `Dict[str, Any]` I think)? I guess since we don't run `mypy` in this repo?